include/grpcpp/impl/codegen/client_callback.h

```diff
@@ -196,14 +237,49 @@ class ClientBidiReactor {
   /// AddHold or AddMultipleHolds before StartCall.  If there is going to be,
   /// for example, a read-flow and a write-flow taking place outside the
   /// reactions, then call AddMultipleHolds(2) before StartCall. When the
-  /// application knows that it won't issue any more Read operations (such as
+  /// application knows that it won't issue any more read operations (such as
   /// when a read comes back as not ok), it should issue a RemoveHold(). It
   /// should also call RemoveHold() again after it does StartWriteLast or
-  /// StartWritesDone that indicates that there will be no more Write ops.
+  /// StartWritesDone that indicates that there will be no more write ops.
   void AddHold() { AddMultipleHolds(1); }
   void AddMultipleHolds(int holds) { stream_->AddHold(holds); }
   void RemoveHold() { stream_->RemoveHold(); }
```

Comment:
Do the number of Removes need to match the number of Holds?  If yes, it should be explicitly stated so.

Reply:
Thanks. Yes it does, and I added that.